repo_utils/include_config.py

```
@@ -0,0 +1,57 @@
+def load_include_patterns(repo_root: Path) -> pathspec.PathSpec | None:
+    """Load .codeboarding-include from .codeboarding/ dir and compile a PathSpec.
+
+    Returns None if the file doesn't exist or contains only comments/blank lines.
+    """
+    include_path = _get_include_path(repo_root)
+    if not include_path.exists():
+        return None
+
+    try:
+        lines = include_path.read_text(encoding="utf-8").splitlines()
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)